torch_ttnn/passes/lowering/target_wrappers.py

```diff
@@ -16,3 +16,71 @@ def repeat(t, sizes):
 @torch.fx.wrap
 def pack_to_tuple(*args):
     return tuple(args)
+
+
+@torch.fx.wrap
+def move_to_host(device_tensor, layout):
+    host_tensor = ttnn.from_device(device_tensor)
+    return ttnn.to_layout(host_tensor, layout)
```

Comment:
Means layout change is on host. In some cases its possible to do layout change on device.
Also, I am not sure if plan to_layout removes tile padding.